### ISSUE TITLE
Need an interface to setup LAN

### ISSUE THREAD

**Issue by moskovich** [apc-io/apc-firefox-os#25]
Inside settings we need an interface to configure common network settings.

**Comment by ferndot** [comment]
@moskovich what repo should we push changes to?  This repository has no files except the readme.

**Comment by nano3k0a** [comment]
Im very confused, maybe because im not so used to github. Where can i find the Soruce Code of this APC Firefox OS Project? How can i take a look at it? Do i already need a APC? More detailed Readme about the Source Code would be very nice. thx.

**Comment by trungnt** [comment]
@Trodis: If you follow the building guide, you will have the source code after the step 2.1.2 at https://github.com/apc-io/apc-firefox-os/blob/master/building-guide.md#212--configure-b2g-for-your-device

**Comment by Tsaoally** [comment]
We need Ethernet setting that enable DHCP or Use Static IP (IP address, Gateway, Netmask, DNS1/DNS2)

**Comment by lechau-apc** [comment]
A porting or implementation e.g. based on an existing popular network managers should be fine. Anyway, all innovation is welcome :)

**Comment by nano3k0a** [comment]
Hi thx for the advise, i got the source code but im getting a weird error. Mabye its a bug, or i did not set up my system the right way. I followed your instruction and followed the instruction from mozilla how to setup my system to build Firefox OS. So when i try to run ./config.sh emulator from mozillas repo or ./config.sh wmdi from the apc repo i get this error: http://pastebin.com/raw.php?i=jVGUgX5n 

I already asked the guys in the IRC Channel but no body was available. Hope u guys can help me. Sorry for bothering and thx for your help.

**Comment by trungnt** [comment]
Hi Trodis, just a dumb question, but do you have git command available in your $PATH?

**Comment by nano3k0a** [comment]
yeah it is, my Path contains this: /usr/local/bin:/usr/bin:/bin:/usr/local/sbin:/usr/sbin:/sbin:/usr/bin/git:/opt/android-sdk/tools:/opt/java/bin:/opt/java/db/bin:/opt/java/jre/bin:/usr/bin/core_perl

**Comment by trungnt** [comment]
Hi Trodis. Please give us some more details like what is your Linux distro, what is your python version. Thanks!

**Comment by nano3k0a** [comment]
My System is Arch x64 and my Python is Python 3.3.2.

**Comment by trungnt** [comment]
Ok, thanks Trodis. Please use Python 2 instead, config.sh uses repo command to sync the repositories and repo is written in Python but not fully support Python 3 yet.

**Comment by kwonjeagu1** [comment]
We need Ethernet setting that enable DHCP or Use Static IP (IP address, Gateway, Netmask, DNS1/DNS2)
